resources/main.mjs

```diff
@@ -78,18 +74,18 @@ class MainBenchmarkClient {
     didFinishLastIteration() {
         console.assert(this._isRunning);
         this._isRunning = false;
-        const results = this._computeResults(this._measuredValuesList, this.displayUnit);
 
-        this._updateGaugeNeedle(results.mean);
-        document.getElementById("result-number").textContent = results.formattedMean;
-        if (results.formattedDelta)
-            document.getElementById("confidence-number").textContent = `\u00b1 ${results.formattedDelta}`;
+        const scoreResults = this._computeResults(this._measuredValuesList, "score");
+        this._updateGaugeNeedle(scoreResults.mean);
+        document.getElementById("result-number").textContent = scoreResults.formattedMean;
+        if (scoreResults.formattedDelta)
+            document.getElementById("confidence-number").textContent = `\u00b1 ${scoreResults.formattedDelta}`;
 
+        const results = this._computeResults(this._measuredValuesList, params.unit);
```

Comment:
The only issue I have with this commit is that I don't know if anyone looking at the detailed view would want to see it displayed as "score" - and this changes the default to do that. My hypothesis is that the details view is only useful for browser engineers - who would all like to see the results in ms. So I wonder if in practice what we want is to have a param like  `showDetails` instead of `unit` which only controls which panel is visible by default and then we always have Summary = score and Details = ms.

I could be convinced otherwise if you feel strongly about this though, and will also ask a few of our engineers for thoughts.

Reply:
humm you're right, there is little added value from having params.unit.
- Let me remove params.unit
- Summary view always shows score
- Detailed view always shows ms